src/main.py

```diff
@@ -1,60 +1,51 @@
-from argparse import ArgumentParser
 from math import ceil, log10
 from os import listdir, path, rename
 from pathlib import Path
 from sys import argv
+from src.utils import parse_args
 
 from natsort import natsorted
 
-
-def parse_args(argv):
-    parser = ArgumentParser(
-        description='Series Renamer: rename files of series with respect to episode number.')
-    parser.add_argument('--directory', '-d', dest='directory',
-                        metavar='[path/to/target/directory]', type=str, required=False, default='.', help='path to target directory')
-    parser.add_argument('--prefix', '-p', dest='prefix',
-                        metavar='[Series Name S01 E]', type=str, required=True, help='name prefix before episode number')
-    parser.add_argument('--dry-run', '-dr', dest='dry_run',
-                        action='store_true', help='dry run')
-
-    return parser.parse_args(argv)
+FIRST_EPISODE_NUMBER = 1
 
 
 def calculate_new_file_name(prefix, full_path, current_episode, episode_precision):
     extension = Path(full_path).suffix
     formatted_episode_number = f"{current_episode}".zfill(
         episode_precision)
-    
     return f"{prefix}{formatted_episode_number}{extension}"
 
 
 def calculate_episode_precision(file_names):
     return ceil(log10(len(file_names) + 1))
 
 
-def main():
-    parsed_args = parse_args(argv[1:])
-    directory = path.realpath(parsed_args.directory)
-    prefix = parsed_args.prefix
-    dry_run = parsed_args.dry_run
+def change_file_name_format(file_name, episode_number, episode_precision, parsed_args):
```

Comment:
```suggestion
def change_file_name(file_name, episode_number, episode_precision, parsed_args):
```

Reply:
Adding a commit with the changes (and adjusting the relevant unit tests as well :))